pmblocklist.php

```diff
@@ -64,7 +63,7 @@
 		MiscFunctions::exitPage(strlen($void)." error(s) occured:<br />".$echo, true);
 	}
 	unset($rec, $user, $ck, $k, $void, $new, $f, $i);
-} elseif($_GET["action"] == "unblock") {
+} elseif($_POST["action"] == "unblock") {
```

Comment:
We'll stick to POST for now, since it can be longer than GET, and less ugly when unblocking many users